src/envoy/http/jwt_auth/jwt_authenticator_test.cc

```diff
@@ -731,6 +757,25 @@ TEST_F(JwtAuthenticatorTest, TestOnDestroy) {
   auth_->onDestroy();
 }
 
+// In the test, there is no forward_payload_header
+class NoForwardPayloadHeaderTest : public JwtAuthenticatorTest {
```

Comment:
Don't need to create new Test class. Just call SetupConfig in your test function to reset config, same as you did with other tests.

Reply:
Done.